constants.go

```diff
@@ -94,15 +94,16 @@ type DescriptorType uint8
 
 // Descriptor types defined by the USB spec.
 const (
-	DescriptorTypeDevice    DescriptorType = C.LIBUSB_DT_DEVICE
-	DescriptorTypeConfig    DescriptorType = C.LIBUSB_DT_CONFIG
-	DescriptorTypeString    DescriptorType = C.LIBUSB_DT_STRING
-	DescriptorTypeInterface DescriptorType = C.LIBUSB_DT_INTERFACE
-	DescriptorTypeEndpoint  DescriptorType = C.LIBUSB_DT_ENDPOINT
-	DescriptorTypeHID       DescriptorType = C.LIBUSB_DT_HID
-	DescriptorTypeReport    DescriptorType = C.LIBUSB_DT_REPORT
-	DescriptorTypePhysical  DescriptorType = C.LIBUSB_DT_PHYSICAL
-	DescriptorTypeHub       DescriptorType = C.LIBUSB_DT_HUB
+	DescriptorTypeDevice      DescriptorType = C.LIBUSB_DT_DEVICE
+	DescriptorTypeConfig      DescriptorType = C.LIBUSB_DT_CONFIG
+	DescriptorTypeString      DescriptorType = C.LIBUSB_DT_STRING
+	DescriptorTypeInterface   DescriptorType = C.LIBUSB_DT_INTERFACE
+	DescriptorTypeCSInterface DescriptorType = 0x24
```

Comment:
what about CSEndpoint?

also add a comment to specify where the value was sourced from. (in this case I assume this is CDC spec v1.1/1.2?), since this descriptor type is not a part of the USB spec as such.